README.md

```diff
@@ -103,12 +98,8 @@ make env
 
 Now that the necessary components are installed, you can pursue by one the following:
 
-1. [Review the campaign workflow and the automation tools](./docs/workflow_overview.md)
-2. [Generate smatch audit list](./docs/generate_smatch_audit_list.md)
-3. [Run kAFL boot and usermode harnesses](./bkc/kafl)
-
-# Targeting your own guest kernel [TBD]
-
-1. Port the provided guest kernel harnesses to your target kernel
-2. Set `$LINUX_GUEST` to your target kernel source tree
-3. Perform the above workflow steps based on your new target kernel
+1. [Review the campaign workflow and the automation tools](docs/workflow_overview.md)
```

Comment:
Got a merge conflict here and above for this PR - I fixed up the ToC to remove the reference to `bkc/kafl` and add new item 5. The readme in bkc/kafl is probably redundant now as well, perhaps something can be recycled.

Reply:
Thanks, the change makes sense, integrated. 

The readme under `bkc/kafl` still have several sections not redundant. I am think about how to update it.